### PR TITLE
Menubar material

### PR DESCRIPTION
**Pull request by goyal-Dushi**
Addressing issue #1059.
Designed the Editor UI using material and CSS 

https://user-images.githubusercontent.com/40212568/124360378-118a5800-dc47-11eb-94a1-6b091593d7bc.mp4

### CODE REVIEW

**Review by AbhipsaGuru1012 — COMMENTED** [review]
@goyal-Dushi there are some changes that I would recommend

1) The image that you have used for line spacing is slightly bigger than the other images that you have used. Please try to make it of height as the other images. 

2) Also the input field where you are showing the line-spacing value, can you expand it a bit to fit all the characters. I have attached an image below that shows the current display.
doc2pen

**Comment by goyal-Dushi** [comment]
Hey @AbhipsaGuru1012, I have added your recommended changes.

File: src/pages/Editor/components/Slider/NumberSlider.jsx [hunk 1]
```diff
@@ -1,9 +1,28 @@
 import styles from "./slider.module.scss";
 import React, { useState } from "react";
+import { Tooltip, Avatar } from "@material-ui/core";
+import { withStyles, makeStyles } from "@material-ui/core";
+
+const CustomTooltip = withStyles({
```
Comment (anushbhatia) [inline src/pages/Editor/components/Slider/NumberSlider.jsx:6]: this should be defined in CSS.

**Review by anushbhatia — CHANGES_REQUESTED** [review]
Double-clicking it goes away. So try resolving that cause if the user sets something and clicks it again it shouldn't go away.

File: src/pages/Editor/components/Slider/NumberSlider.jsx (hunk 1 above)
Comment (goyal-Dushi) [reply src/pages/Editor/components/Slider/NumberSlider.jsx:6]: Hey, @anushbhatia, these are all material imports, thus, they are imported within the component jsx. Check below for the usage.
https://material-ui.com/getting-started/usage/(url)

**Comment by goyal-Dushi** [comment]
> Double-clicking it goes away. So try resolving that cause if the user sets something and clicks it again it shouldn't go away.

Actually, I have made it so that the user could toggle through it. The user is supposed to click the option once in order to get the option to set values. If you want the option to remain after a single click then no problem, I can add so.

**Comment by smaranjitghose** [comment]
This_should

This should not overlap.

When I assigned the issue, I expected a redesign of the complete menubar and not replacement of certain text with material icons

**Comment by goyal-Dushi** [comment]
Hey, @smaranjitghose, I see you have closed the pull request, I can still customize the layout if you want and the overlap issue if you are accepting the pull request.

**Pull request closed without merging**